google/cloud/security/common/gcp_api/cloud_resource_manager.py

```diff
@@ -114,16 +114,17 @@ def get_project_iam_policies(self, resource_name, project_numbers):
         """
         projects_stub = self.service.projects()
 
-        try:
-            for project_number in project_numbers:
+        for project_number in project_numbers:
```

Comment:
The logic to loop through project numbers should be handled in the calling function, not here. This function should just try to get the IAM policy for a single project, either by project_id or project_number. Either works when used as the resource in getIamPolicy, so make sure the docstring and arg name is generic. The function can just return the response then if there are no errors, no need to have it be a generator or create a dictionary for the response.

This function should still raise APIExecutionError on exception, and then the calling function can decide what to do when an exception is raised. (Log an error, set a flag in the database record, etc).

Otherwise if when you mask the exception inside of this loop, the caller has no way of knowing it, or you need to create a new key in your response dict, which is messy and not very Pythonic.